SparkPost/2web2ui
# FE-911: Remove Suppressions Banner on Dashboard

### What Changed
 - Removed Suppression Banner

### How To Test
 - Create a new account age it by a week (can't really do)
 - Open the dashboard to confirm suppression banner is not displayed
